index.js

```diff
@@ -1,118 +1,235 @@
 /************************************************************** Task 1: Warm-up! **************************************************************/
 //Task a: declare a variable called votingAge, console log true if age > 18 (no function required)
+let votingAge=20;
+if(votingAge > 18){
+    console.log(true)
+}
 
 
-
+// What I want to answer
 
 
 //Task b: declare a variable and then use a conditional to change the value of that variable based on the value assigned to a second variable (no function required)
 
-
-
+let dog = "Sofie";
+let dogFur = "red";
+if (dogFur != "Fluffy Black & white"){
+    dog = "Defintely not Sofie";
+}
+console.log (dog)
 
 
 //Task c: Convert string ("1999") to integer (1999)  (no function required) // hint look up the Number method
 
+console.log(parseInt("1999"));
 
 
 
 
-//Task d: Write a function to multiply a*b 
-
-
+//Task d: Write a function to multiply a*b
 
+function multiply(a, b){
+    return a*b;
+}
+console.log(multiply(10, 20))
 
 
 /************************************************************** Task 2 **************************************************************/
 //Age in Dog years
-//write a function that takes your age and returns it to you in dog years - they say that 1 human year is equal to seven dog years 
-
-
-
+//write a function that takes your age and returns it to you in dog years - they say that 1 human year is equal to seven dog years
 
+let ageInDogYears;
+let humanAge;
+let ageConverter = function(humanAge) {
+    ageInDogYears = humanAge * 7;
+    console.log("I am " + ageInDogYears + " in dog years.");
+    }
+    ageConverter(32);
+// returning undefined for some reason... grrr
```

Comment:
Because you didn't return anything. You logged it to the output but didn't return anything.